doc/classes/@GlobalScope.xml

```diff
@@ -788,9 +788,9 @@
 				printraw("A")
 				printraw("B")
 				printraw("C")
-				# Prints ABC
+				# Prints ABC to terminal
```

Comment:
![image](https://user-images.githubusercontent.com/85438892/202579967-c0151e0a-1119-4ac4-b435-def848dfcc59.png)

To me both way seems right.

Reply:
The docs like to omit articles to shorten the text length sometimes. That's one of those times. It would be more correct to say "to the terminal" since you're not limited by space up there.

Reply:
For the record, the comments in code snippets have a low documentation value, because they cannot be translated and thus aren't majorly useful to a lot of readers. All the necessary and correct information should be in the surrounding text. In this case the sole reason for the comment is to show that consecutive calls would be executed in order. I don't think the extended note is warranted here, as this should be already communicated by the method's description.

Reply:
I agree with this. Updated the PR to make this clear in the main description.

Reply:
> For the record, the comments in code snippets have a low documentation value, because they cannot be translated and thus aren't majorly useful to a lot of readers.

FYI, comments in code snippets *can* be translated. The whole `<description>` is the `msgid` that translators have to translate. So they also need to be careful not to break the code snippets (which sometimes happens when translators without much Godot experience start translating API calls...).